packages/admin/cms-admin/src/pages/pageTree/PageVisibility.tsx

```diff
@@ -59,7 +60,7 @@ const PageVisibility = ({ page }: PageVisibilityProps): React.ReactElement => {
                     slug: page.slug,
                 },
             },
-        });
+        }).then(() => client.refetchQueries({ include: ["Pages"] }));
```

Comment:
You can do that simpler:

```suggestion
            refetchQueries: ["Pages"],
        });
```